app/controllers/passwordless/sessions_controller.rb

```diff
@@ -37,11 +39,13 @@ def create
     #   or _root_path_
     # @see ControllerHelpers#sign_in
     # @see ControllerHelpers#save_passwordless_redirect_location!
-    def show
+    def show # rubocop:disable Metrics/MethodLength, Metrics/AbcSize
```

Comment:
Oh, and move the comment above the method (or above the docs if necessary)

Reply:
and done